ydb/core/kqp/ut/query/kqp_query_ut.cpp

```diff
@@ -1782,6 +1782,165 @@ Y_UNIT_TEST_SUITE(KqpQuery) {
             CompareYson(output, R"([[1u;[1];["test1"]];[100u;[100];["test2"]]])");
         }
     }
+
+
+    void RunQuery (const TString& query, auto& session, bool expectOk = true) {
+        auto qResult = session.ExecuteQuery(query, NYdb::NQuery::TTxControl::NoTx()).ExtractValueSync();
+        if (!qResult.IsSuccess()) {
+            Cerr << "Query failed, status: " << qResult.GetStatus() << ": " << qResult.GetIssues().ToString() << Endl;
+        }
+        UNIT_ASSERT(qResult.IsSuccess() == expectOk);
+    };
+
+    void ListDir(TKikimrRunner& kikimr, const TString& expectedEntry, const TString& unexpected, bool& found) {
```

Comment:
ListDir is generic function but is specific for topics. Move type of object to params please.